analyzer/file/png/PNG.spicy

```diff
@@ -0,0 +1,194 @@
+module PNG;
+
+import spicy;
+#import Exif;
+
+public type File = unit {
+	signature: bytes &size=8; # has to be 137 80 78 71 13 10 26 10
+	var colour_type: uint8 = 0;
+	: Chunk(self)[] &eod;
+
+	on signature {
+		local d: spicy::Base64Stream;
+		local thesig : bytes = spicy::base64_decode(d, b"iVBORw0KGgo=");
```

Comment:
Is this to get binary data in there? You can do `b"\xYY"`, would that work here? And they maybe just with a `&requires` attribute for the field?

Reply:
This is no longer in there - I changed this to your suggestion.